src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -459,6 +459,10 @@ export class WorkspaceSavedObjectsClientWrapper {
                 [WorkspacePermissionMode.LibraryRead, WorkspacePermissionMode.LibraryWrite]
               ),
             },
+            // By declaring workspaces as null,
+            // workspaces won't be appended automatically into the options.
+            // or workspaces can not be found because workspace object do not have `workspaces` field.
+            workspaces: null,
```

Comment:
if workspaces param is not needed, why do we set it here?

Reply:
We set workspaces as null explicitly to make sure the workspace_id_consumer wrapper won't append workspaces into the query.